projects/meta_cl/experiments/metacl_track_stats.py

```diff
@@ -53,7 +65,42 @@ class TrackStatsMetaCLExperiment(mixins.TrackRepresentationSparsityMetaCL,
 )
 
 
+def plot_entropy_distribution_(_, winning_mask, targets):
```

Comment:
This pattern of using _ for unused arguments looks strange when used in the function definition. Is this a common thing used elsewhere? Maybe we should name the argument even if not using it in the code below.

Reply:
Fair, I'll adjust it.